packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.tsx

```diff
@@ -306,7 +308,7 @@ export const YppAuthorizationModal: FC<YppAuthorizationModalProps> = ({
       {
         text: `Your YouTube channel has at least ${formatNumber(
           fetchedChannelRequirements?.MINIMUM_SUBSCRIBERS_COUNT || 0
-        )} subscribers`,
+        )} subscriber${fetchedChannelRequirements?.MINIMUM_SUBSCRIBERS_COUNT !== 1 ? 's' : ''}`,
```

Comment:
We have a helper function for this `pluralizeNoun`. We could use this here. Not a biggie though